src/app/modules/main/wallet_section/networks/view.nim

```diff
@@ -105,18 +115,19 @@ QtObject:
         networkEnabledToUxEnabledState(n.enabled, allEnabled)
       ))
 
-    self.all.setItems(items)
-    self.layer1.setItems(items.filter(i => i.getLayer() == 1))
-    self.layer2.setItems(items.filter(i => i.getLayer() == 2))
-    self.enabled.setItems(items.filter(i => i.getIsEnabled()))
+    let filteredItems = items.filter(i => i.getIsTest() == self.areTestNetworksEnabled)
+    self.flatNetworks.setItems(items)
+    self.all.setItems(filteredItems)
+    self.layer1.setItems(filteredItems.filter(i => i.getLayer() == 1))
```

Comment:
Would it make sense to have these `1` and `2` values defined as constants / enum if their meaning are being a layer 1 or layer 2 type? Indeed maybe there is already a `layer type`enum being used in other code places? Since it seems a common and more used concept.

Reply:
adding it here app_service/service/network/dto.nim, I checked it not already added